conduct.md

```diff
@@ -11,8 +11,50 @@
 * Attendance
 * Deadlines
 * Communication
-* Participation
-* Decisions
+
+### Participation
+
+Detailed paragraph describing the category. (delete when added paragraph)
+
+* **Acceptable Conduct (A):**
+  * *Members shall*
+    * Points here
+
+* **Unacceptable Conduct (U):**
+  * *Members shall not*
+    * Points here
+
+### Decisions
+
+A majority of people in a group should agree upon a given decision before it's
+implemented. Decisions should be made on the basis of supporting evidence that show
+that the choice would be reasonable and productive. When unsure about making a decision
+it is a good idea to defer to a TA or the professor. Before enacting a decision you
+should reach out to other groups whom the choice you made will affect. Teams will
+determine how long a decision will take to implement. Group decisions won't be
+foreboding, but should unite the group to solve a problem together!
+
+* **Acceptable Conduct (A):**
+  * *Members shall*
+    * Take into account everyones input in the decision making process.
+    * Stay flexible and be willing to change if it would be productive.
+    * Report errors or bugs to the appropriate team members.
+    * Record what decision was made and when it was implemented.
+    * Defer to the Professor or teaching assistants when concerned about a decision.
+    * Use online polls to reach a group consensus about a decision.
+    * Team members should decide the priority of the issue using a "URGENT" tag.
```

Comment:
Is it the priority of the team issues or in general all the issues? And does this need to go under decision section since it's a decision made about "issue tags"?